community-operators/aqua/aqua-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,460 @@
+              "port": 3622
+            },
+            "sendingHostImages": false,
+            "runcInterception": false
+          }
+        },
+        {
+          "apiVersion": "operator.aquasec.com/v1alpha1",
+          "kind": "AquaGateway",
+          "metadata": {
+            "name": "aqua"
+          },
+          "spec": {
+            "requirements": false,
+            "serviceAccount": "aqua-sa",
+            "dbSecretName": "aqua-database-password",
+            "dbSecretKey": "db-password",
+            "deploy": {
+              "replicas": 1,
+              "service": "ClusterIP",
+              "image": {
+                "repository": "gateway",
+                "registry": "registry.aquasec.com",
+                "tag": "3.5",
+                "pullpolicy": "IfNotPresent"
+              }
+            },
+            "aquaDb": "aqua-database",
+            "ssl": false,
+            "auditSsl": false,
+            "openshift": false
+          }
+        },
+        {
+          "apiVersion": "operator.aquasec.com/v1alpha1",
+          "kind": "AquaScanner",
+          "metadata": {
+            "name": "aqua"
+          },
+          "spec": {
+            "requirements": false,
+            "serviceAccount": "aqua-sa",
+            "deploy": {
+              "replicas": 1,
+              "service": "ClusterIP",
+              "image": {
+                "repository": "scanner",
+                "registry": "registry.aquasec.com",
+                "tag": "4.0",
+                "pullPolicy": "IfNotPresent"
+              }
+            },
+            "login": {
+              "username": "administrator",
+              "password": "Password1",
+              "host": "http://aqua-server-svc:8080"
+            },
+            "openshift": false
+          }
+        },
+        {
+          "apiVersion": "operator.aquasec.com/v1alpha1",
+          "kind": "AquaServer",
+          "metadata": {
+            "name": "aqua"
+          },
+          "spec": {
+            "requirements": false,
+            "add": false,
+            "serviceAccount": "aqua-sa",
+            "dbSecretName": "aqua-database-password",
+            "dbSecretKey": "db-password",
+            "rbac": {
+              "enable": true,
+              "openshift": false,
+              "privileged": true
+            },
+            "deploy": {
+              "replicas": 1,
+              "service": "LoadBalancer",
+              "image": {
+                "repository": "console",
+                "registry": "registry.aquasec.com",
+                "tag": "3.5",
+                "pullPolicy": "IfNotPresent"
+              }
+            },
+            "aquaDb": "aqua-database",
+            "adminPassword": "Password1",
+            "licenseToken": null,
+            "clusterMode": true,
+            "ssl": false,
+            "auditSsl": false,
+            "dockerless": false,
+            "openshift": false
```

Comment:
Should this be true? `community-operators` is the directory for OpenShift deployed operators. Also apply this to all other CRs with this field

Reply:
yes, its also for kubernetres then we have the option

Reply:
The `comunity-operators` directory is for OpenShift deployed operators, so if this field should be 'true' when used in OpenShift, then it should be true in this directory

Reply:
Updated